client/src/containers/overview/project-details/index.tsx

```diff
@@ -0,0 +1,386 @@
+    { name: "Social feasibility", rating: "Medium" },
+    { name: "Security risk", rating: "Medium" },
+    { name: "Value for money", rating: "Low" },
+    { name: "Overall", rating: "Medium" },
+  ],
+  costEstimates: [
+    {
+      name: "Capital expenditure",
+      value: 1514218,
+      items: [
+        { name: "Feasibility analysis", value: 70000 },
+        { name: "Conservation planning and admin", value: 629559 },
+        { name: "Data collection and field costs", value: 76963 },
+        { name: "Community representation", value: 286112 },
+        { name: "Blue carbon project planning", value: 111125 },
+        { name: "Establishing carbon rights", value: 296010 },
+        { name: "Validation", value: 44450 },
+        { name: "Implementation labor", value: 0 },
+      ],
+    },
+    {
+      name: "Operating expenditure",
+      value: 36509571,
+      items: [
+        { name: "Monitoring and Maintenance", value: 402322 },
+        { name: "Community benefit sharing fund", value: 34523347 },
+        { name: "Carbon standard fees", value: 227875 },
+        { name: "Baseline reassessment", value: 75812 },
+        { name: "MRV", value: 223062 },
+      ],
+    },
+    { name: "Total cost", value: 38023789 },
+  ],
+};
+
+interface IProjectDetailsProps {
+  projectName: string;
+  open: boolean;
+  setIsOpen: (open: boolean) => void;
+}
+
+export default function ProjectDetails({
+  projectName,
+  open,
+  setIsOpen,
+}: IProjectDetailsProps) {
+  return (
+    <Sheet open={open} onOpenChange={setIsOpen}>
+      <SheetContent className="overflow-y-scroll sm:max-w-[50%]">
+        <SheetHeader className="space-y-6">
+          <div className="flex gap-4">
+            <div className="flex items-center gap-2">
+              <Button
+                variant="outline"
+                className="h-8 w-8 p-0 hover:bg-transparent"
+              >
+                <ChevronDown className="h-4 w-4" />
+              </Button>
+              <Button
+                variant="outline"
+                className="h-8 w-8 p-0 hover:bg-transparent"
+              >
+                <ChevronUp className="h-4 w-4" />
+              </Button>
+            </div>
+            <div className="flex items-center justify-between">
+              <SheetTitle>{projectName}</SheetTitle>
+            </div>
+          </div>
+          <ParametersProjects />
+        </SheetHeader>
+
+        <div className="mt-6 space-y-6">
+          <div className="grid grid-cols-2 gap-4">
+            <Card className="flex flex-col gap-4 p-4">
+              <div className="flex flex-col gap-2">
+                <div className="flex flex-col gap-2">
+                  <div className="flex items-center gap-2">
+                    <h3 className="text-md font-medium">Total project cost</h3>
+                    <InfoButton title="Refers to the summary of Capital Expenditure and Operating Expenditure" />
+                  </div>
```

Comment:
There's a`Label` component with a `tooltip` property where you can achieve the same result, please use it.